src/main/kotlin/com/devfloor/untitled/hashtag/application/HashtagService.kt

```diff
@@ -8,7 +8,6 @@ import org.springframework.stereotype.Service
 class HashtagService(
     private val repository: HashtagRepository,
 ) {
-    fun createByName(hashtagName: String): Hashtag {
-        return repository.findByName(hashtagName) ?: repository.save(Hashtag(hashtagName))
-    }
+    fun createByName(hashtagName: String): Hashtag =
+        repository.findByName(hashtagName) ?: repository.save(Hashtag(hashtagName))
```

Comment:
```suggestion
    fun createByName(hashtagName: String): Hashtag = repository.findByName(hashtagName)
            ?: repository.save(Hashtag(hashtagName))
```

이거 이렇게 할 수도 있나? 내가 아직 코틀린 문법이 어색해서 그런건지.. 기존 삼항연산자랑 비슷하게 생겨서 그런가..
이게 더 이해가 잘 가긴 하던데 한번 참고해주면 좋을 듯 ㅎㅅㅎ